### ISSUE TITLE
Problems with the new update 3.10

### ISSUE THREAD

**Issue by Ragolay** [popcorntime/popcorntime#284]
Operating System Version:  Windows 7 Ultimate

Popcorn Time Version:
 0 3.10

Download date:
 I install Popcorn on Febraury, but today I see the new update, but Avast! Antivirus does not allow me certain decissions, and I had to uninstall the previous version and install the 3.10 with the Avast! Antivirus disabled.

#### Expected Behaviour
 I think that the 3.10 would be better than the previous version (logic).

#### Actual Behaviour
 The 3.10 has the following errors: the window of Popcorn is not minimized when I select another window (Firefox, e.g.) in the Windows toolbar; and the synopses are not translated.

#### Steps to reproduce the behaviour

1. Open Popcorn Time.
2. Try to open Firefox (e. g.) while Popcorn is in "first plane".
3. Observe that Firefox is expanded behind Popcorn.

**Comment by Ragolay** [comment]
What more info can I offer? :D

**Comment by VitorVRS** [comment]
Can you print? Can you show what Avast are not allowing you to do?

**Comment by Ragolay** [comment]
I can not show it because finally I could install the 3.10, but...when I tried install the 3.10, Avast! sent a message saying something like "the file is for an unknown author, we will send a report to ensure". I disabled Avast! and installed the 3.10, I reactivated Avast! and later, I received a message from Avast! saying something like "the file passed positively recognition".

Therefore, I could solve the issue of Avast. The problem are the other two: no translated synopsis; and when another program is pressed on the Windows toolbar, the window expands "behind" Popcorn, but Popcorn not pass to being behind the pressed.

Sorry if something is not understood, my English level is limited :P

Thanks for try help me :)

PD: the problem with the subtitles is solved :D
